main.qml

```diff
@@ -42,7 +42,7 @@ import "wizard"
 
 ApplicationWindow {
     id: appWindow
-
+    title: "Monero GUI"
```

Comment:
What it should be is the actual name of the piece of software, like Monero Prime or Monero Kerna. But the community hasn't yet come to a conclusion about what the name for the GUI should be. See discussion here: https://github.com/monero-project/monero-core/issues/674